src/core_atmosphere/Registry.xml

```diff
@@ -3452,10 +3452,10 @@
                 <var name="ter" type="real" dimensions="nCells" units="m"
                      description="terrain height"/>
 
-                <var name="albedo12m" type="real" dimensions="nMonths nCells" units="unitless"
+                <var name="albedo12m" type="real" dimensions="nMonths nCells" units="percent"
                      description="monthly-mean climatological surface albedo"/>
 
-                <var name="greenfrac" type="real" dimensions="nMonths nCells" units="unitless"
+                <var name="greenfrac" type="real" dimensions="nMonths nCells" units="percent"
                      description="monthly-mean climatological greeness fraction"/>
```

Comment:
For consistency with the `init_atmosphere` core's `Registry.xml` file, shall we update the spelling of "greeness" here as well?

Reply:
of course. done.